dev/config/monitored-pod/configmap.yaml

```diff
@@ -4,7 +4,7 @@ data:
     <!DOCTYPE html>
     <html>
     <head>
-    <title>Welcome to nginx!</title>
+    <title>Welcome-updated-updated-updated-updated to nginx!</title>
```

Comment:
@pratikjagrut I believe this is committed by accident. Let's revert this. This file is updated during the k8s-test, but it shouldn't be committed.

Reply:
Oh yes.